src/measure_theory/set_integral.lean

```diff
@@ -78,6 +82,16 @@ begin
   rwa [inter_eq_self_of_subset_left h] at this
 end
 
+lemma measurable_on.comp (hs : is_measurable s) (hf : measurable_on s f) (hgm : measurable g) :
```

Comment:
For composition, forming `g ∘ f`, the assumptions on `g` should come first in the statement. Also, this one should be called `measurable.comp_measurable_on`. The lemma `measurable_on.comp` should rather say that, if `g` is measurable on a set `t`, and `f` is measurable on a set `s`, and `s` is contained in the preimage of `t` under `f`, then `g ∘ f` is measurable on `s`. You can have a look at the interface of `continuous_on` or `differentiable_on`: if we can have a coherent naming scheme, it will help a lot!